README.md

```diff
@@ -20,8 +20,14 @@ export DB_USER= <...>
 export DB_PASSWORD= <...>
 ```
 
+###Cron job for clearning 1 day old authkeys
+
+`0 * * * * ~/swecc-server/venv/bin/python /swecc-server/server/manage.py prune_authkeys`
```

Comment:
perhaps a way to add this as part of lifecyle of `runserver` (start/stop job on init/destroy)? Or we can just make it a step in Dockerfile too if that's annoying